### ISSUE TITLE
Only partial support for mod: tiny progressions

### ISSUE THREAD

**Issue by SwordShaman** [TheXFactor117/Levels#113]
It seems spears added by the mod are given your levels, but battle axes are not.  I'm not sure what system you use for supporting weapons added by other mods.  Could you possibly add a whitelist to the config in which we can manually add specific, missed items?

**Comment by TheXFactor117** [comment]
Yes, I will look into that when I get a chance. It's something that needs to be worked on, though I think it might be up to me to provide support for other mods.
